src/_P042_Candle.ino

```diff
@@ -76,422 +83,440 @@ enum ColorType {
   ColorSelected
 };
 
-uint8_t Candle_red = 0;
-uint8_t Candle_green = 0;
-uint8_t Candle_blue = 0;
-uint8_t Candle_bright = 128;
-SimType Candle_type = TypeSimpleCandle;
-ColorType Candle_color = ColorDefault;
+uint8_t   Candle_red    = 0;
+uint8_t   Candle_green  = 0;
+uint8_t   Candle_blue   = 0;
+uint8_t   Candle_bright = 128;
+SimType   Candle_type   = TypeSimpleCandle;
+ColorType Candle_color  = ColorDefault;
 
 // global variables
 unsigned long Candle_Update = 0;
-word Candle_Temp[4] = { 0, 0, 0 };     // Temp variables
-int Candle_Temp4 = 0;
-boolean GPIO_Set = false;
+word Candle_Temp[4]         = { 0, 0, 0 }; // Temp variables
```

Comment:
Why an array of 4 and only 3x a 0?

Reply:
Fixed, was wrongly declared as 4, should have been 3.